Makefile

```diff
@@ -53,10 +53,19 @@ FUZZTIME ?= "10s"
 all: install build
 .PHONY: all
 
+
+
+
 install:
 	git lfs install
 	go mod download
+
+ifeq ($(OS),Windows_NT)
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANGCI_LINT_VERSION)
+else
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s $(GOLANGCI_LINT_VERSION)
+endif
```

Comment:
This shouldn't be necessary we do not want to use different method of installation for a different OS 😕

Reply:
I did that, because using the installation from the source is not recommended by the  golangci-lint team. [LINK](https://golangci-lint.run/welcome/install/#install-from-source) And that is a bug that only happens on the windows shell. But we can adopt the **go install** to all OS. What you prefer @fasmat ?

Reply:
We could skip installing `golangci-lint` at all on the windows runners since it is not needed outside of the `lint` workflow step that is only executed on the `ubuntu-22.04` runner.

In a second step (not needed for this PR) we can install `golangci-lint` on `make lint` if it is not installed yet and not have a `make install` target any more.

Reply:
The issue to me seems to be clearly the runner and not our `Makefile` - I reported the issue also here:

https://github.com/actions/runner-images/issues/10009